src/inactive-blog-filter.js

```diff
@@ -0,0 +1,145 @@
+/**
+ * Returns whether or not the blog should be marked as inactive
+ * Criteria for filtering (in milliseconds) is based on filters.json under blog.inactive
+ */
+const fs = require('fs');
+const dotenv = require('dotenv');
+const feedParser = require('./feed-parser');
+
+// Configure the environment
+dotenv.config();
+
+/**
+ * Condition for passing redlist some() check
+ * @param {string} feedUrl - url of the feed to check against redlist
+ * @param {Object} redItem - object containing a redlist entry
+ * @returns {boolean} - bool value
+ */
+function isRedlisted(feedUrl, redItem) {
+  return (feedUrl === redItem.url);
+}
+
+/**
+ * Simple function that takes the difference between current date and post date
+ * If difference is greater than threshold, the blog is considered inactive
+ * @param {Date} postDate - javascript date object of the post being checked
+ * @returns {number} - date difference in milliseconds
+ */
+function dateDiff(postDate) {
+  const currentDate = new Date();
+  return currentDate - postDate;
+}
+
+/**
+* Callback for redlist check
+* @callback checkCallback
+* @param {boolean} result - true/false whether or not a feed url is redlisted
+*/
+
+/**
+ * Checks if feed url is redlisted
+ * @param {string} feedUrl - url of the feed to check against redlist
+ * @param {checkCallback} callback - a callback that runs after the check
+ */
+function check(feedUrl, callback) {
+  // Read redlist file
+  fs.readFile('feeds-redlist.json', (err, redListRaw) => {
+    if (err) {
+      // Error reading file
+      console.warn('problem reading redlist file:', err.message);
+      callback(false);
```

Comment:
In node, you always propagate the error back as the first arg.  In this case, `callback(err)`

Reply:
I will address this in the next PR push.